http/vibe/http/common.d

```diff
@@ -300,7 +300,7 @@ class HTTPResponse {
 		InetHeaderMap headers;
 
 		/// All cookies that shall be set on the client for this request
-		@property ref DictionaryList!Cookie cookies() { return m_cookies; }
+		@property scope ref DictionaryList!Cookie cookies() return { return m_cookies; }
```

Comment:
```suggestion
		@property ref DictionaryList!Cookie cookies() scope return { return m_cookies; }
```
?

Reply:
Do we even need the `scope` at all here? I still couldn't force myself to look through the spec to fully grasp the `scope` additions (assuming that everything is correctly documented by now) and from the observed effects I'm unsure whether `return` a) implies `scope`, b) is required to be used in conjunction with `scope`, or c) has an independent effect on the parameter/`this` pointer.

Reply:
We need `scope` only if we intend to call this function with a `scope DictionaryList!Cookie` is my understanding. But I have a similar issue with the spec.